pkg/cmd/target/drop/options_test.go

```diff
@@ -0,0 +1,35 @@
+/*
+SPDX-FileCopyrightText: 2021 SAP SE or an SAP affiliate company and Gardener contributors
+
+SPDX-License-Identifier: Apache-2.0
+*/
+
+package drop_test
+
+import (
+	"github.com/gardener/gardenctl-v2/internal/util"
+	. "github.com/gardener/gardenctl-v2/pkg/cmd/target/drop"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/gomega"
+)
+
+var _ = Describe("Options", func() {
+	It("should validate", func() {
+		streams, _, _, _ := util.NewTestIOStreams()
+		o := NewOptions(streams)
+		o.Kind = TargetKindGarden
+		o.TargetName = "foo"
+
+		Expect(o.Validate()).To(Succeed())
+	})
+
+	It("should reject invalid kinds", func() {
+		streams, _, _, _ := util.NewTestIOStreams()
+		o := NewOptions(streams)
+		o.Kind = TargetKind("not a kind")
+		o.TargetName = "foo"
+
+		Expect(o.Validate()).NotTo(Succeed())
```

Comment:
also check for the error message, at least if it contains a certain string like `invalid target kind given, must be one of`

Reply:
I tried to do so, but wasn't able to extract the error message from errOut. Maybe because we don't run the command?